pom.xml

```diff
@@ -1581,9 +1581,9 @@
     </profile>
 
     <profile>
-      <id>hadoop-2.4</id>
```

Comment:
Does this imply that we'd be dropping support for Hadoop 2.4?  I don't think we can do that.